tests/integration/workflow.py

```diff
@@ -244,3 +257,41 @@ def run(self, *args, **kwargs):
             )
         )
         futures.wait(future)
+
+
+class WorkflowWithTooBigData(Workflow):
+    name = "example"
+    version = "example"
+    task_list = "example"
+
+    def run(self, *args, **kwargs):
+        group = Chain(break_on_failure=False)
+        group.append(pingpong_activity, "ab", JUMBO_FIELDS_MAX_SIZE)
+        group.append(pingpong_activity, "ab" * JUMBO_FIELDS_MAX_SIZE, 0.25)
+        fut = self.submit(group)
+        futures.wait(fut)
+        print(f"Result: {fut.result} Exception: {fut.exception}")
+
+
+class PingPingWorkflow(Workflow):
+    name = "pingpong"
+    version = "example"
+    task_list = "example"
+
+    def run(self, d, n):
+        return pingpong(d, n)
+
+
+class WorkflowWithTooBigDataInWorkflow(Workflow):
```

Comment:
Can we at least name this `InSubworkflow` ? It took me half a minute trying to see what was different from the test above.